src/KubernetesClient.Basic/KubernetesClient.Basic.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFrameworks>netstandard2.0</TargetFrameworks>
+        <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
would this cause VS issue?
seems my VS works well

Reply:
Yes. I've isolated this change locally and restarting VS solved the bogus errors problem.

Reply:
Possibly related to https://github.com/dotnet/sdk/issues/8676?